### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by mkb93**
backend with a few tests

### CODE REVIEW

File: api/package.json
Comment (veritywong) [inline api/package.json:1]: does link 2 and link 9 need changing to no longer reference acebook?

File: api/bin/www
Comment (veritywong) [inline api/bin/www:1]: does this reference to acebook need to be changed?

**Review by veritywong — COMMENTED** [review]
There are some references to acebook that may need to be updated to flock

**Review by veritywong — APPROVED** [review]

**Review by ev-th — APPROVED** [review]
Legendary

**Pull request merged**